src/veins/base/phyLayer/MacToPhyControlInfo.h

```diff
@@ -1,8 +1,7 @@
 #pragma once
 
 #include "veins/veins.h"
-
```

Comment:
removing this empty line squashes two #include blocks together. I'd rather keep three separate blocks in .h files (standard library includes, veins.h, and the rest).